--- libs/octomap-1.6.8/CMakeLists.txt
+++ libs/octomap-1.6.8/CMakeLists.txt
@@ -3,8 +3,8 @@ PROJECT( octomap-distribution )
 
 ENABLE_TESTING()  # enable CTest environment of subprojects
 
-option(BUILD_OCTOVIS_SUBPROJECT "Build targets from subproject octovis" ON)
-option(BUILD_DYNAMICETD3D_SUBPROJECT  "Build targets from subproject dynamicEDT3D" ON)
+option(BUILD_OCTOVIS_SUBPROJECT "Build targets from subproject octovis" OFF)
+option(BUILD_DYNAMICETD3D_SUBPROJECT  "Build targets from subproject dynamicEDT3D" OFF)
 
 ADD_SUBDIRECTORY( octomap )
 
